test/controllers/phylo_trees_controller_test.rb

```diff
@@ -0,0 +1,35 @@
+require 'test_helper'
+
+class PhyloTreesControllerTest < ActionDispatch::IntegrationTest
+  setup do
+    @user = users(:one)
+    post user_session_path, params: { 'user[email]' => @user.email, 'user[password]' => 'password' }
+    @project = projects(:one)
+  end
+
+  test 'should get index' do
+    get "/phylo_trees/index"
+    assert_response :success
+  end
+
+  test 'should get new' do
+    taxid = 573
+    get "/phylo_trees/new?taxid=#{taxid}&project_id=#{@project.id}"
```

Comment:
wait will this fail if 573 already exists?

Reply:
no